index.html

```diff
@@ -67,24 +67,21 @@ <h2>let's connect</h2>
           </a>
         </li>
         <li>
-          <a href="#">
+         
             <img src="./images/Vector.png" alt="vector">
 
     </div>
   </section>
   </section>
-
-<br>
-
     <!-- works -->
 
     
 <div class="works">
```

Comment:
- [ ] Kindly as requested by the previous reviewer consider wrapping the entire section in a section tag not wrapping each work card in a section tag I think you've got the previous reviewer in a wrong way 
-  here is a hint :
```
<section class="works">
  
<div class="card-works">
 <div class="first-image">
  <img src="./images/Snapshoot Portfolio (1).png" alt="">
 </div>
  
 <h2 class="headings">Tonics</h2>

 <ul class="first-item">
   <li class="one"> <a href="#">Canopy</a> </li>
   <li class="other"> <a href="#">back end dev</a></li>
   <li class="other"> <a href="#">2015</a></li>
 </ul>

  <p class="text-1">
    A daily selection of privately <br>
     personalized reads; no accounts or sign-ups required.
  </p>

  <ul class="item-2">
    <li class="link-item"> <a href="#"></a> html</li>
    <li  class="link-item" > <a href="#">css</a></li>
    <li class="link-item"> <a href="#">javascript</a></li>

  </ul>

   <p class="see">
    <a href="#">See projects</a>
   </p>

  </div>
add other divs too .................................
</section>
```

this is for semantics purposes as it helps in improving your website SEO